index.d.ts

```diff
@@ -546,6 +546,64 @@ console.log(stdout);
 export function execaCommand(command: string, options?: Options): ExecaChildProcess;
 export function execaCommand(command: string, options?: Options<null>): ExecaChildProcess<Buffer>;
 
+type Expression = string | number | Array<string | number>;
```

Comment:
Small nit-picking: what do you think about naming this type more specifically. `Expression` is rather generic, and I did not realize at first it was meant only for the template strings expressions under `${...}`. Maybe `TemplateExpression`, or anything else more specific?

Reply:
Good call! `TemplateExpression` is 👌 https://github.com/sindresorhus/execa/pull/510/commits/6ed5b89f7d37f15e707bed0239788ef2555e9f45